kiwi/boot/image/dracut.py

```diff
@@ -58,6 +62,75 @@ def include_file(self, filename, install_media=False):
             self.included_files_install.append('--install')
             self.included_files_install.append(filename)
 
+    def include_module(self, module):
+        """
+        Include module to dracut boot image
+
+        :param string module: module to include
+        """
+        if module not in self.modules:
+            self.modules.append(module)
+            self.modules_args = [
+                '--add', ' {0} '.format(' '.join(self.modules))
+            ]
+
+    def omit_module(self, module):
+        """
+        Omit module to dracut boot image
+
+        :param string module: module to omit
+        """
+        if module not in self.omit_modules:
+            self.omit_modules.append(module)
+            self.omit_modules_args = [
+                '--omit', ' {0} '.format(' '.join(self.omit_modules))
+            ]
+
+    def clear_modules_config(self):
+        """
+        Clears the modules configuration
+        """
+        self.modules = []
+        self.modules_args = []
+        self.omit_modules = []
+        self.omit_modiles_args = []
+
+    def persistent_config(self, config=None, config_file=None):
```

Comment:
The name of the method is imho not telling what action is behind the call. I suggest something like

* write_system_config
* create_system_config
* create_persistent_config
* write_system_config_file

Something in that direction. Personally I like write_system_config_file()

Reply:
renamed to write_system_config_file